### ISSUE TITLE
Error: Unhandled 'error' event

### ISSUE THREAD

**Issue by kopireng06** [BetaHuhn/deploy-to-vercel-action#393]
**🐞 Describe the bug**

Cannot deploy to vercel because there an is error happened

**🖼️ Screenshots**

![image](https://github.com/user-attachments/assets/b7afe639-aaf6-48c2-a608-acf3c9625f4b)

**⚙️ Environment**
- Action version: v1.0.0

**Comment by Septiven** [comment]
facing the same issue here.

**Comment by evereq** [comment]
Yes, same issue for me, it seems something changed in Vercel so this action basically stop working :(

**Comment by JoaoVSouto** [comment]
A workaround that is working for me is to add a previous step to install the Vercel CLI:

```yaml
- name: Install Vercel CLI
  run: npm install -g vercel@22.0.1
```

**Comment by treygilliland** [comment]
I also ran into this and can confirm @JoaoVSouto's workaround fixed it

**Comment by jonystorm** [comment]
same problem here, tried @JoaoVSouto solution worked great, thanks!

**Comment by winsrewu** [comment]
Emmm, have you guys checked this? https://[github.com/actions/runner-images/issues/10636](https://github.com/actions/runner-images/issues/10636)
They removed vercel cli.

Which means maybe we need a quick update before they upgrade the runner on 1/17?

**Comment by jlison** [comment]
It just stopped working for me. I tried:
```
    runs-on: ubuntu-20.04
```
and
```
- name: Install Vercel CLI
  run: npm install -g vercel@22.0.1
```
with no luck. Any suggestions on alternative workarounds?

**Comment by winsrewu** [comment]
@jlison 
try runs-on: ubuntu-latest

**Comment by vincentdchn** [comment]
Same problem here, tried @JoaoVSouto workaround and it did the job, thx

**Comment by romanmesh** [comment]
same problem. any suggestions?

**Comment by vincentdchn** [comment]
@romanmesh Did you try the workaround from @JoaoVSouto ?

**Comment by wottpal** [comment]
I think at this point this repository should be classified as unmaintained.

@mountainash Does that issue also exist in [your fork](https://github.com/mountainash/deploy-to-vercel-action) or could you fix it there? I guess a lot of people would be happy to collectively declare yours as the new official one if you are still interested :P

**Comment by mountainash** [comment]
It would depend on all the parameters conditioning the use of the script, but I'm using my fork to deploy about 3 websites multiple times a day without issues - so I guess it's working.

There [was communications a while ago](https://github.com/BetaHuhn/deploy-to-vercel-action/pull/384#issuecomment-2094848737) to migrate/publicise my fork more - but I've stalled on this as I plan to be offline for nearly a year from next month - and won't be around to keep the project in good shape. I'll pick this up again with @BetaHuhn in 2026. So for now; choose what works - or fork your own (with PRs if you make it better please).

**Comment by parkerthegeniuschild** [comment]
@JoaoVSouto is the 🐐

**Comment by zognotadog** [comment]
Started facing this / related issues to this again even with @JoaoVSouto's fix. 

Bumping to v48 fixed it.

```
- name: Install Vercel CLI
   run: npm install -g vercel@48.0.0
```
